openapi.yml

```
@@ -0,0 +1,383 @@
+      requestBody:
+        description: Request body parameters of an organziation notification POST request
+        required: true
+        content:
+          application/json:
+            schema:
+              allOf:
+                - $ref: '#/components/schemas/NotificationRequest'
+                - type: object
+                  properties:
+                    organization_id:
+                      $ref: '#/components/schemas/NotificationId'
+                  required: [organization_id]
+  /notifications/member:
+    post:
+      summary: Send notification to a specific member of org
+      operationId: post-notification-member
+      responses:
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Notification'
+      requestBody:
+        description: Request body parameters of a member notification POST request
+        required: true
+        content:
+          application/json:
+            schema:
+              allOf:
+                - $ref: '#/components/schemas/NotificationRequest'
+                - type: object
+                  properties:
+                    member_id:
+                      $ref: '#/components/schemas/NotificationId'
+                  required: [member_id]
+  /email:
+    post:
+      summary: Send email to an email address
+      operationId: post-email
+      responses:
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Notification'
+      requestBody:
+        description: Request body parameters of an email POST request
+        required: true
+        content:
+          application/json:
+            schema:
+              allOf:
+                - $ref: '#/components/schemas/NotificationRequest'
+                - type: object
+                  properties:
+                    email_address:
+                      type: string
+                      format: email
+                  required: [email_address]
+  /notifications/{notification_id}:
+    get:
+      summary: Get information about a specific notification
+      operationId: get-notification
+      responses:
+        '200':
+          description: Notification information
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Notification'
+      parameters:
+        - $ref: '#/components/parameters/NotificationId'
+    delete:
+      summary: Cancel a notification
+      operationId: delete-notification
+      responses:
+        '200':
+          description: Updated
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Notification'
+      parameters:
+        - $ref: '#/components/parameters/NotificationId'
+  /preferences/org/{organization_id}:
```

```suggestion
  /preferences/organization:
```

Source the organization_id from the JWT